src/components/app/index.js

```diff
@@ -1,95 +1,31 @@
-import { createContext, useState } from 'react';
+import { useReducer } from 'react';
 import {
+  DeleteModal,
+  FilmCardInformation,
+  FilmFormModule,
   HeaderPanel,
   MainPage,
-  FilmFormModule,
-  SuccessModal,
-  DeleteModal,
-  FilmCardInformation
+  SuccessModal
 } from '../index';
-import { filmList, defaultFilm } from '../../constants/const';
-import './App.css';
+import { defaultState, init } from 'context/context';
+import { reducer } from 'context/reducer';
+import { AppContext } from 'context/context';
```

Comment:
Can rewrite to import from one package by adding index.js file under ./src/context and import all variables from all files under 'context'
```suggestion
import { AppContext, defaultState, reducer, init } from 'context';
```